stories/Form.js

```diff
@@ -146,6 +148,61 @@ stories
       size={select('Size', ['', 'small', 'medium', 'large'], '') || undefined}
     />
   ))
+  .add('Combobox', () => (
+    <Combobox
+      inlineListbox={boolean('Inline listbox variant', false)}
+      readOnly={boolean('Read-only variant', false)}
+      error={text('Error', '') || undefined}
+      height={select('Height', [5, 7, 10]) || undefined}
```

Comment:
Returns a `string` but should return a `number` for the height.

Reply:
This is a limitation of the storybook knobs addon. Can't do much about it at the moment.